misc/scripts/build.sh

```diff
@@ -123,7 +123,7 @@ then
   $DOCKER run --rm  -v $PWD/..:$dir -w $dir $pylint_container $MAKE_ESX pylint
 else
   docker_socket=/var/run/docker.sock
-  if [ -z '$SSH_KEY_OPT' ]
+  if [ -z "$SSH_KEY_OPT" ]
```

Comment:
we need to add some reference in the contributing.md about when to initialize `SSH_KEY_OPT ` and how does it look like for any contributor to set while using the key which stores on non-default path.

Reply:
+1
We need to add one line into the doc about how to set SSH_KEY_OPT.
It needs to be set in the format of:
export SSH_KEY_OPT='-i /Users/username/.ssh/id_rsa'
The key point is using '' instead of "".

Reply:
Done.